src/pages/BugReportForm.scss

```diff
@@ -0,0 +1,81 @@
+// BugReportForm.scss
+
+.bug-report-form-container {
+  max-width: 800px;
```

Comment:
```suggestion
```

Reply:
If you want to remove the max-width, you also need to set

```suggestion
  max-width: 800px;
```
But I don't think it looks good. 
For users with wider screen like 24''/27'' it will look like this: 

![image](https://github.com/hasadna/open-bus-map-search/assets/107395990/238be399-c683-477e-82b3-b339d704bfd0)

So in my opinion, we should set a limit to the width.